consensus/core/src/test_dag.rs

```diff
@@ -40,17 +40,20 @@ pub(crate) fn build_dag(
             .collect::<Vec<_>>(),
     };
 
+    let num_authorities = context.committee.size();
     let starting_round = ancestors.first().unwrap().round + 1;
     for round in starting_round..=stop {
         let (references, blocks): (Vec<_>, Vec<_>) = context
             .committee
             .authorities()
             .map(|authority| {
                 let author_idx = authority.0.value() as u32;
-                let base_ts = round as BlockTimestampMs * 1000;
+                // Test the case where a block from round R+1 can have small timestamp than a block from round R.
```

Comment:
nit: "have a smaller"

Also can you add a todo to port this behavior over to the dag builder/parser as we will eventually remove these test helper functions.